manifests/concourse-manifest/spec/manifest_generation_spec.rb

```diff
@@ -34,18 +33,10 @@ def merge_fixtures(fixtures)
     ).to eq("https://" + fixtures["terraform_outputs"]["concourse_dns_name"])
   end
 
-  it "gets values from generated secrets" do
-    # fixture_atc_password = fixtures["jobs"][0]["templates"][2]["properties"]["basic_auth_password"]
-
-    expect(
-      manifest_with_defaults["jobs"][0]["templates"][1]["properties"]["basic_auth_password"]
-    ).to eq(fixtures["secrets"]["concourse_atc_password"])
-  end
-
-  it "gets values from predefined secrets" do
+  it "gets values from secrets" do
     expect(
       atc_template.fetch("properties").fetch("basic_auth_password")
```

Comment:
Out of interest, are `manifest_with_defaults["jobs"][0]["templates"][1]["properties"]["basic_auth_password"]` (old test) and `atc_template.fetch("properties").fetch("basic_auth_password")` (new test) the same thing?

I couldn't tell from 26ca5f043cab05df5b7c4b8f06a49a8c9c226685 either.